pycsw/core/util.py

```diff
@@ -220,10 +220,39 @@ def bbox2wktpolygon(bbox):
 
     if bbox.startswith('ENVELOPE'):
         bbox = wktenvelope2bbox(bbox)
-    minx, miny, maxx, maxy = [float(coord) for coord in bbox.split(",")]
-    return 'POLYGON((%.2f %.2f, %.2f %.2f, %.2f %.2f, %.2f %.2f, %.2f %.2f))' \
+    minx, miny, maxx, maxy = [format_coord(float(coord)) for coord in bbox.split(",")]
+    wktGeometry = 'POLYGON((%s %s, %s %s, %s %s, %s %s, %s %s))' \
         % (minx, miny, minx, maxy, maxx, maxy, maxx, miny, minx, miny)
+    return wktGeometry
 
+def format_coord(number):
+    """Return desired precision of coordinate
+    Uses the env "COORDINATE_PRECISION" to set the precise number of digits after dot (for desired accurate result).
+    If the env is not specified, the default precise number is 2 digits after the dot.
+
+    Parameters
+    ----------
+    number: float
+        A number presenting part of the coordinate
```

Comment:
`presenting` -> `representing`.
Even better (IMO):
`A number presenting part of the coordinate` -> `A coordinate single value`